### PR TITLE
Added second bool with option destroy

### PR DESCRIPTION
**Pull request by sstanytska**
Hello, everyone!
I added second bool with option Terraform Destroy.
Please see my changes file and let me know if something wrong.

Thank you!

### CODE REVIEW

File: src/com/lib/JenkinsDeployerPipeline.groovy [hunk 1]
```diff
@@ -23,10 +23,11 @@ def runPipeline() {
         print('This branch does not supported')
   }
 
-  node('master') {
+  node('dev') {
```
Comment (fsadykov) [inline src/com/lib/JenkinsDeployerPipeline.groovy]: Node means one of the Jenkins slave. I am note sure why you change to dev

File: src/com/lib/JenkinsDeployerPipeline.groovy [hunk 2]
```diff
@@ -67,6 +68,25 @@ def runPipeline() {
...
+      stage('Terraform Destroy') {
+
+        if (params.terraformDestroy) {
+
+          dir("${WORKSPACE}/deployment/terraform") {
+            echo "##### Terraform Destroing ####"
+            sh "terraform destroy"
+          }
+
+        } else {
```
Comment (fsadykov) [inline src/com/lib/JenkinsDeployerPipeline.groovy]: I think we can exclude here.  You have to also make sure when user select terraform apply, terrafom destroy shouldn't work.

File: src/com/lib/JenkinsDeployerPipeline.groovy (hunk 1 above)
Comment (sstanytska) [reply src/com/lib/JenkinsDeployerPipeline.groovy]: of course.. I don't need to change node name... Thank you!

File: src/com/lib/JenkinsDeployerPipeline.groovy (hunk 2 above)
Comment (sstanytska) [reply src/com/lib/JenkinsDeployerPipeline.groovy]: I have to put here only terraform destroy?

**Pull request merged**